Mathlib/Algebra/Group/End.lean

```diff
@@ -665,22 +665,22 @@ theorem inv_def (e₁ : MulAut M) : e₁⁻¹ = e₁.symm :=
   rfl
 
 @[simp]
-theorem mul_apply (e₁ e₂ : MulAut M) (m : M) : (e₁ * e₂) m = e₁ (e₂ m) :=
-  rfl
+theorem inv_symm (e : MulAut M) : e⁻¹.symm = e := rfl
```

Comment:
Do you really want this modification? `inv_symm` already exists, and I don't see why removing `mul_apply`.

Reply:
This is a mistake, I believe.

Reply:
(The merge was painful.)

Reply:
Ah no, the `mul_apply` is below.

Reply:
So it looks OK, it's just that the `inv` lemmas are together, and the `apply` as well.